mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+====================================================
+Customizable unsubscribe link on mass mailing emails
+====================================================
+
+With this module you can set a custom unsubscribe link append at bottom of mass
+mailing emails.
+
+
+Configuration
+=============
+
+To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
+and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
+of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+
+For example:
+```
```

Comment:
Thanks for your point, I've also found this online parser: http://rst.ninjs.org/
Very usefull to check if everything is correct, because it shows up errors and warnings